integrations/nlu-snips/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@jovotech/nlu-snips",
+  "version": "4.0.0-alpha.1",
+  "description": "",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm5/index.js",
+  "es2015": "dist/esm2015/index.js",
+  "types": "dist/types/index.d.ts",
+  "sideEffects": false,
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "prebuild": "tsc -b tsconfig.build.cjs.json tsconfig.build.esm5.json tsconfig.build.esm2015.json tsconfig.build.types.json --clean",
+    "build": "tsc -b tsconfig.build.cjs.json tsconfig.build.esm5.json tsconfig.build.esm2015.json tsconfig.build.types.json",
+    "watch": "tsc -b tsconfig.build.cjs.json tsconfig.build.esm5.json tsconfig.build.esm2015.json tsconfig.build.types.json --watch",
+    "prettier": "prettier -w -l src test",
+    "eslint": "eslint src test --fix --ext .ts",
+    "rimraf": "rimraf dist",
+    "test": "jest --runInBand"
+  },
+  "author": "jovotech",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@jovotech/framework": "^4.0.0-alpha.16",
+    "@types/jest": "^26.0.20",
+    "@types/node": "^10.17.50",
+    "@typescript-eslint/eslint-plugin": "^4.12.0",
+    "@typescript-eslint/parser": "^4.12.0",
+    "eslint": "^7.17.0",
+    "eslint-config-prettier": "^7.1.0",
+    "eslint-plugin-prettier": "^3.3.1",
+    "jest": "^26.6.3",
+    "prettier": "^2.2.1",
+    "rimraf": "^3.0.2",
+    "ts-jest": "^26.5.4",
+    "typescript": "^4.2.3"
+  },
+  "peerDependencies": {
+    "@jovotech/framework": ">=4.0.0-alpha.16 <4.0.0"
+  },
+  "gitHead": "9d39800224d277cd6b500b48ba3f0fbacb22f176",
+  "prettier": "../../.prettierrc.js",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
```

Comment:
This should be above `devDependencies` in terms of the order to stay consistent with the other package.json-files.

Reply:
It might not be clear in Github. I mean the `dependencies`.

Reply:
Done :+1: